modules/olm-mirroring-catalog-colocated.adoc

```diff
@@ -45,7 +46,9 @@ $ oc adm catalog mirror \
 <4> Optional: If you do not want to configure trust for the target registry, add the `--insecure` flag.
 <5> Optional: Specify which platform and architecture of the index image to select when multiple variants are available. Images are passed as `'<platform>/<arch>[/<variant>]'`. This does not apply to images referenced by the index. Valid values are `linux/amd64`, `linux/ppc64le`, `linux/s390x`, `linux/arm64`.
 <6> Optional: Generate only the manifests required for mirroring without actually mirroring the image content to a registry. This option can be useful for reviewing what will be mirrored, and lets you make any changes to the mapping list, if you require only a subset of packages. You can then use the `mapping.txt` file with the `oc image mirror` command to mirror the modified list of images in a later step. This flag is intended for only advanced selective mirroring of content from the catalog.
+--
 +
+--
```

Comment:
This looks fine in the output, but I'm wondering if the example output block should be part of the previous open block (so that there is one open block for all content within step 2). It probably doesn't matter since it won't affect the output.